ansible/roles/commcarehq/tasks/main.yml

```diff
@@ -110,3 +110,26 @@
   tags:
     - localsettings
     - hq-localsettings
+
+- name: Copy celery restart script
+  sudo: yes
+  template:
+    src: "restart_celery.sh.j2"
+    dest: "/root/restart_celery.sh"
```

Comment:
This is not very importat, but just to keep things tidy, this should either go to /usr/local/sbin, if for some reason, we prefer to keep them at root's home, they should go in /root/bin/